tests-integration/cli/cli.test.ts

```diff
@@ -0,0 +1,123 @@
+            "logic: String -> String",
+            "logic level =",
+            `   String.append "Player level: " level`
+        ))
+
+        const IR = await cli.make(PATH_TO_PROJECT, CLI_OPTIONS)
+        const rentalsModule = IR.distribution[3].modules[0]
+        expect(rentalsModule[1].value.types).toMatchObject([])
+    })
+
+    test("should create an IR with no values when no values are found in elm file", async () => {
+        await writeFile(path.join(PATH_TO_PROJECT, 'src/Package', 'Rentals.elm'), join(
+            "module Package.Rentals exposing (Action)",
+            "",
+            "type Action",
+            `   = Rent`,
+            `   | Return`
+        ))
+
+        const IR = await cli.make(PATH_TO_PROJECT, CLI_OPTIONS)
+        const rentalsModule = IR.distribution[3].modules[0]
+        expect(rentalsModule[1].value.values).toMatchObject([])
+    })
+
+    test("should create an IR with both types and values", async () => {
+        await writeFile(path.join(PATH_TO_PROJECT, 'src/Package', 'Rentals.elm'), join(
+            "module Package.Rentals exposing (..)",
+            "",
+            "type Action",
+            `   = Rent`,
+            `   | Return`,
+            "",
+            "logic: String -> String",
+            "logic level =",
+            `   String.append "Player level: " level`
+        ))
+        const IR = await cli.make(PATH_TO_PROJECT, CLI_OPTIONS)
+        const rentalsModule = IR.distribution[3].modules[0]
+        expect(rentalsModule[1].value.values).not.toMatchObject([])
+        expect(rentalsModule[1].value.types).not.toMatchObject([])
+    })
+
+    test("should create an IR with only types when typesOnly is set to true", async () => {
+        await writeFile(path.join(PATH_TO_PROJECT, 'src/Package', 'Rentals.elm'), join(
+            "module Package.Rentals exposing (..)",
+            "",
+            "type Action",
+            `   = Rent`,
+            `   | Return`,
+            "",
+            "logic: String -> String",
+            "logic level =",
+            `   String.append "Player level: " level`
+        ))
+        const IR = await cli.make(PATH_TO_PROJECT, { typesOnly: true })
+        const rentalsModule = IR.distribution[3].modules[0]
+        expect(rentalsModule[1].value.values).toMatchObject([])
+        expect(rentalsModule[1].value.types).not.toMatchObject([])
+    })
+
+    afterAll(async () => {
+        await rmdir(path.join(__dirname, 'temp'), { recursive: true })
+    })
+})
+
+
+describe("Testing Morphir-elm make with expected IR", () => {
+    test("should generate the expected IR", async () => {
+        const projectDir = path.join(__dirname, 'test-data', 'rentals')
+
+        const ir = await cli.make(projectDir, { typesOnly: false })
+        let file = await readFile(path.join(projectDir, 'expected-morphir-ir.json'))
+        const expectedJSON = JSON.parse(file.toString())
+
+        expect(ir).toStrictEqual(expectedJSON)
+    })
+
+})
```

Comment:
I think testing specific paths in the IR will be more maintainable then expecting the whole IR to match a preset value. Let's remove this.

Reply:
ok sure